### PR TITLE
Branch 3: Reorganize data into folders for each data set

### PR DESCRIPTION
**Pull request by thomaslchan**
- Tiles fail to load?

### CODE REVIEW

**Comment by thomaslchan** [comment]
Don't merge this

File: fake-files/output-expected/wang/wang.images.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+ "MERrfish": {
+  "sample": 1,
+  "tileSource": "https://s3.amazonaws.com/vitessce-data/0.0.18/reorganize_folders/mermaid.images/info.json"
```
Comment (mccalluc) [inline fake-files/output-expected/wang/wang.images.json:4]: `mermaid.images` shouldn't be here, right?

File: python/loom_reader.py [hunk 2]
```diff
@@ -41,7 +41,8 @@ def clusters(self):
         '''
         Returns information about each cluster.
 
-        >>> lr = LoomReader('fake-files/input/linnarsson.cells.loom')
+        >>> lr = LoomReader('fake-files/input/linnarsson/\
+linnarsson.cells.loom')
```
Comment (mccalluc) [inline python/loom_reader.py:45]: slight preference just to turn off the line length check for this section.

**Review by mccalluc — APPROVED** [review]
Still having a reference to `mermaid.images` in `wang.images.json` seems odd, and we should just modify the linting to allow longer lines. but I'll go ahead and merge.

**Pull request merged**